src/main/java/io/github/ashisbored/playerpronouns/PlayerPronouns.java

```diff
@@ -191,4 +191,9 @@ public static boolean setPronouns(ServerPlayerEntity player, @Nullable Pronouns
 
         return true;
     }
+
+    public static @Nullable Pronouns getPronouns(ServerPlayerEntity player) {
```

Comment:
It's probably worth exposing a second version of this which takes a UUID directly too